lms/templates/instructor/instructor_dashboard_2/membership.html

```diff
@@ -233,9 +238,11 @@
       data-rolename="Moderator"
       data-display-name="${_("Discussion Moderators")}"
       data-info-text="
-        ${_("Discussion moderators can edit or delete any post, clear misuse flags, close "
+        ${_("Discussion Moderators can edit or delete any post, clear misuse flags, close "
          "and re-open threads, endorse responses, and see posts from all cohorts. "
-         "They CANNOT add/delete other moderators and their posts are marked as 'staff'.")}"
+         "Their posts are marked as 'staff'. They cannot manage course team membership by "
+         "adding or removing discussion moderation roles. You can only give course team "
+         "roles to enrolled users.")}"
       data-list-endpoint="${ section_data['list_forum_members_url'] }"
       data-modify-endpoint="${ section_data['update_forum_role_membership_url'] }"
       data-add-button-label="${_("Add Moderator")}"
```

Comment:
(this is a button label, and one of three that relate to roles for discussions. if I change this one, I think I should change the other two as well, but since the UI already shows what the full name of the role is elsewhere, adding it to the buttons just seems unnecessary)